src/helpers/customHooks/useWindowSize.js

```diff
@@ -12,7 +12,7 @@ function useWindowSize() {
     function handleResize() {
       // Set window width/height to state
       setWindowSize({
-        width: window.innerWidth,
+        width: window.screen.width,
```

Comment:
Hi, why was this change required? Since the innerWidth. Since the window.screen.width would be constant when the window Width changed. This could add extra issues with the responsiveness. and using a hook for a constant might be inefficient

Reply:
window.innerWidth was giving very different values on some screen sizes. I remember, on the screen size that was around 956px, this was giving a very different width. This was the actual reason that sidebar was not opening on some small screen sizes.

I have checked the window.screen.width is giving the exact correct width on all screen sizes and due to that bug of the sidebar not opening on some screen sizes got resolved automatically. (Because the code on the sidebar for responsiveness was absolutely correct but since this window.innerWidth was giving wrong output that's why that issue was persisting.

I have checked almost in all components and this change is not affecting any of them negatively.

Reply:
It could be the solution but it is a bit inefficient. either directly use window.screen.width for now on the places that is required, since the window.screen.width stays the same for a device. useWindowSize() hook was used to handle places when the window size changes

Reply:
I have done the changes.  I am using window. screen.width only in the sidebar component now,